packages/react-components/src/components/Form/Form.tsx

```diff
@@ -3,8 +3,6 @@ import {
   FormProps as ReactAriaFormProps,
 } from "react-aria-components";
 
-export type FormProps = ReactAriaFormProps;
```

Comment:
Can you please make sure we are exporting the RAC `FormProps` in the Form barrel file at `src/components/Form/index.ts`? Otherwise this will break like Checkbox did in 485.

Reply:
Fixed in [cb8eea1](https://github.com/bcgov/design-system/pull/483/commits/cb8eea1aa913b081cdbe26b540ec7b340783601a)